### PR TITLE
SQ-51/Fix navigation to details, tweak cards

### PR DESCRIPTION
**Pull request by rock3r**
This PR:

 * restores navigating to event details when tapping in the schedule
 * animates the card elevation when tapped
 * slightly tweaks the card top padding as per specs
 * fixes excessive margin on speakerless items

 Before | After
 --- | ---
 fix-and-stuff | card-tweaks

details

### CODE REVIEW

File: app/src/main/java/net/squanchy/navigation/Navigator.java [hunk 1]
```diff
@@ -1,11 +1,29 @@
...
 
-    void toSearch();
+    private final Context context;
 
+    public Navigator(Context context) {
+        this.context = context;
+    }
+
+    public void up() {
+        // No-op (top level yo)
```
Comment (tiwiz) [inline app/src/main/java/net/squanchy/navigation/Navigator.java:18]: Why preserve the empty method?

File: app/src/main/res/values/attrs.xml [hunk 2]
```diff
@@ -20,6 +20,7 @@
     <item name="android:background">@drawable/card_background</item>
     <item name="android:foreground">@drawable/card_touch_feedback</item>
     <item name="android:elevation">@dimen/card_elevation</item>
+    <item name="android:stateListAnimator">@anim/card_state_list_anim</item>
```
Comment (tiwiz) [inline app/src/main/res/values/attrs.xml:23]: Shouldn't we keep the style into a `styles.xml` files, instead of the `attrs.xml`?

Comment (rock3r) [reply app/src/main/res/values/attrs.xml:23]: You keep asking, I keep answering, no, this is a default view style.

File: app/src/main/java/net/squanchy/navigation/Navigator.java (hunk 1 above)
Comment (rock3r) [reply app/src/main/java/net/squanchy/navigation/Navigator.java:18]: Navigation is very much WIP, can't guarantee what will happen here. Just not touching unrelated stuff for now.

**Review by tiwiz — APPROVED** [review]

**Pull request merged**